pkg/controllers/dataexport/reconcile.go

```
@@ -1332,6 +1324,33 @@ func (c *Controller) cleanUp(driver drivers.Interface, de *kdmpapi.DataExport) e
 		if err != nil && !k8sErrors.IsNotFound(err) {
 			return fmt.Errorf("delete %s job: %s", de.Status.TransferID, err)
 		}
+		//TODO : Need better way to find BL type from de CR
+		// For now deleting unconditionally for all BL type.
+
+		// secret, err := core.Instance().GetSecret(de.Name, de.Spec.Source.Namespace)
```

Can we remove the comment code and have one TODO comment alone.
I feel, we can go ahead with the approach of delete them unconditionally and if it is NotFound, we can ignore the error and continue.

---

Sure Will do